Automattic/wp-calypso
# Post Type List: Reduce PostItem Margins

This PR brings the left/right margins of post items in the condensed post list inline with the test and icons of the section navigation. It also fixes an IE11 bug where the min-height was causing post titles to be vertically top aligned, instead of center.

**Before:**
<img width="917" alt="before" src="https://user-images.githubusercontent.com/942359/32801719-dc558a72-c94b-11e7-8a8a-2f78a05bfe06.png">

**After:**
<img width="919" alt="no-min-height" src="https://user-images.githubusercontent.com/942359/32801725-e2454da0-c94b-11e7-9d1e-422e036f9b4e.png">

**To test:**
- Checkout branch or run from calypso.live link.
- Add yourself to the `condensedPost` variant of the `condensedPostList` AB test.
- Navigate to `/posts/` and verify that the post item margins are inline with the section nav - 16px all around.
